### PR TITLE
Improve format handling in GStreamer caps

### PR DESCRIPTION
**Pull request by petr-nechaev**
Explicit specification of formats in capabilities improves user experience.
NV16 support is crucial for converting video from VIN modules.

### CODE REVIEW

**Comment by petr-nechaev** [comment]
Hello @ywakasawa , @dhobsong . Could you please merge this pull request?

**Comment by dhobsong** [comment]
Hi @petr-nechaev .  Thank you for these patches. 
I don't have write access for this repo and @ywakasawa will be on vacation until next week.
I had some minor comments though, so I will add those inline.

File: gst/vspfilter/gstvspfilter.c [hunk 1]
```diff
@@ -64,7 +64,12 @@ enum
   PROP_OUTPUT_IO_MODE,
 };
 
-#define CSP_VIDEO_CAPS GST_VIDEO_CAPS_MAKE (GST_VIDEO_FORMATS_ALL)
+#define CSP_VIDEO_CAPS \
+    "video/x-raw, " \
+        "format = (string) {RGB16, RGB, BGR, ARGB, xRGB, BGRA, BGRx, I420, NV12, UYVY}," \
+        "width = [ 1, 8192 ], " \
+        "height = [ 1, 8192 ], " \
```
Comment (dhobsong) [inline gst/vspfilter/gstvspfilter.c]: According to the spec sheet, the maximum width and height is 8190, not 8192.

Comment (dhobsong) [reply gst/vspfilter/gstvspfilter.c]: Is the "(experimental)" tag here needed?  Offhand I can't see any reason why this shouldn't work. Have you noticed some potential problems with the NV16 support?

**Review by dhobsong — CHANGES_REQUESTED** [review]

**Comment by petr-nechaev** [comment]
Thank you for accepting changes. Sorry for delay - had no time to fix the patch

**Pull request closed without merging**